src/Microsoft.AspNet.Mvc.Core/ActionResults/PartialViewResult.cs

```diff
@@ -50,10 +51,22 @@ public class PartialViewResult : ActionResult
             var viewEngine = ViewEngine ??
                              context.HttpContext.RequestServices.GetRequiredService<ICompositeViewEngine>();
 
+            var logger = context.HttpContext.RequestServices.GetRequiredService<ILoggerFactory>()
+                .CreateLogger<PartialViewResult>();
+
             var viewName = ViewName ?? context.ActionDescriptor.Name;
-            var view = viewEngine.FindPartialView(context, viewName)
-                                 .EnsureSuccessful()
-                                 .View;
+            var viewEngineResult = viewEngine.FindPartialView(context, viewName);
+            if (!viewEngineResult.Success)
+            {
+                logger.LogError(
+                    "The partial view '{PartialViewName}' was not found. Searched locations: {SearchedViewLocations}",
+                    viewName,
+                    viewEngineResult.SearchedLocations);
+            }
+
+            var view = viewEngineResult.EnsureSuccessful().View;
+
+            logger.LogVerbose("The partial view '{PartialViewName}' was found successfully.", viewName);
```

Comment:
Though now that I think about it - do we even want to log successful things? What value does that add? I'm thinking that if someone is trying to investigate an issue they probably just want to see failures and other suspicious items. By logging all kinds of successful things (parameter was model bound, type conversion succeeded, filter was run) you could end up with an awful lot of data to sift through.